README.md

```diff
@@ -16,6 +16,10 @@ source code if you already have it handy), and it will figure out all of that
 file's dependencies and sub-dependencies, passing an array of them to your
 callback. Or use the `Sync` API to get the results as a return value.
 
+Result array has non-enumerable `patterns` property with an array of globs;
+each of the found globs is joined with every path progeny looks for dependencies in.
```

Comment:
Ah, now I see. Yes, it's worth explaining that behavior. How about this:

> Glob patterns are joined with each base path progeny searches for dependencies in.

Although is this really desirable behavior? Seems like it could potentially yield some odd results. What if it only returned the paths where matches were found for the pattern?